src/components/main/TreeModal.js

```diff
@@ -24,26 +28,36 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const { useItem, useItems, useOwnItems, useChildren } = hooks;
+const { useItem, useItems, useOwnItems, useChildren, useSharedItems } = hooks;
 
 const TreeModal = ({ itemIds, open, title, onClose, onConfirm, prevent }) => {
   const { t } = useTranslation();
   const classes = useStyles();
-  const { data: ownItems, isLoading } = useOwnItems();
+  const { data: ownItems, isLoading: isOwnItemsLoading } = useOwnItems();
+  // bug: get only shared items with write/admin rights
```

Comment:
I am not sure if the bug consists in `useSharedItems` only fetching shared items with write / admin rights (while it should in fact fetch all of them) or if it actually is the expected behaviour (and instead it fetches too many).

Reply:
It is the intended behavior, but in this context it will result in a bug. I will change with `todo` it will be clearer indeed.